boards/xtensa/esp32/esp32-wrover-kit/configs/lua/defconfig

```diff
@@ -0,0 +1,80 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_ARCH_LEDS is not set
+# CONFIG_ESP32_SPI2_DMA is not set
+# CONFIG_MMCSD_HAVE_CARDDETECT is not set
+# CONFIG_MMCSD_HAVE_WRITEPROTECT is not set
+# CONFIG_MMCSD_MMCSUPPORT is not set
+# CONFIG_NSH_ARGCAT is not set
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+# CONFIG_NSH_CMDPARMS is not set
+# CONFIG_SPI_EXCHANGE is not set
+CONFIG_ARCH="xtensa"
+CONFIG_ARCH_BOARD="esp32-wrover-kit"
+CONFIG_ARCH_BOARD_COMMON=y
+CONFIG_ARCH_BOARD_ESP32_WROVERKIT=y
+CONFIG_ARCH_CHIP="esp32"
+CONFIG_ARCH_CHIP_ESP32=y
+CONFIG_ARCH_CHIP_ESP32WROVER=y
+CONFIG_ARCH_STACKDUMP=y
+CONFIG_ARCH_XTENSA=y
+CONFIG_BOARD_LOOPSPERMSEC=16717
+CONFIG_BUILTIN=y
+CONFIG_DEBUG_ERROR=y
+CONFIG_DEBUG_FEATURES=y
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_MEMCARD=y
+CONFIG_DEBUG_MEMCARD_ERROR=y
+CONFIG_DEBUG_MEMCARD_INFO=y
+CONFIG_DEBUG_MEMCARD_WARN=y
+CONFIG_DEBUG_WARN=y
+CONFIG_ESP32_SPI2=y
+CONFIG_ESP32_SPI2_CSPIN=13
+CONFIG_ESP32_SPI2_MISOPIN=2
+CONFIG_ESP32_SPI2_MOSIPIN=15
+CONFIG_ESP32_UART0=y
+CONFIG_FAT_LCNAMES=y
+CONFIG_FAT_LFN=y
+CONFIG_FS_FAT=y
+CONFIG_FS_PROCFS=y
+CONFIG_HAVE_CXX=y
+CONFIG_HAVE_CXXINITIALIZE=y
+CONFIG_IDLETHREAD_STACKSIZE=3072
+CONFIG_INIT_ENTRYPOINT="nsh_main"
+CONFIG_INTELHEX_BINARY=y
+CONFIG_MMCSD=y
+CONFIG_MMCSD_SPICLOCK=4000000
+CONFIG_MM_REGIONS=3
+CONFIG_NSH_ARCHINIT=y
+CONFIG_NSH_BUILTIN_APPS=y
+CONFIG_NSH_FILEIOSIZE=512
+CONFIG_NSH_LINELEN=64
+CONFIG_NSH_MMCSDSPIPORTNO=2
+CONFIG_NSH_READLINE=y
+CONFIG_PREALLOC_TIMERS=4
+CONFIG_RAM_SIZE=114688
+CONFIG_RAM_START=0x20000000
+CONFIG_RR_INTERVAL=200
+CONFIG_SCHED_WAITPID=y
+CONFIG_START_DAY=6
+CONFIG_START_MONTH=12
+CONFIG_START_YEAR=2011
+CONFIG_SYSTEM_NSH=y
+CONFIG_UART0_SERIAL_CONSOLE=y
+CONFIG_EXAMPLES_LEDS=y
+CONFIG_USERLED_LOWER=y
+CONFIG_USERLED=y
+CONFIG_INTERPRETERS_LUA=y
+CONFIG_EXAMPLES_LUA_MODULE=y
+CONFIG_ARCH_BUTTONS=y
+CONFIG_ARCH_IRQBUTTONS=y
+CONFIG_ESP32_GPIO_IRQ=y
+CONFIG_EXAMPLES_BUTTONS=y
+CONFIG_INPUT=y
+CONFIG_INPUT_BUTTONS=y
+CONFIG_INPUT_BUTTONS_LOWER=y

```

Comment:
Could you check if this works correctly with your project ?

```suggestion
#
# This file is autogenerated: PLEASE DO NOT EDIT IT.
#
# You can use "make menuconfig" to make any modifications to the installed .config file.
# You can then do "make savedefconfig" to generate a new defconfig file that includes your
# modifications.
#
# CONFIG_ARCH_LEDS is not set
# CONFIG_ESP32_SPI2_DMA is not set
# CONFIG_MMCSD_HAVE_CARDDETECT is not set
# CONFIG_MMCSD_HAVE_WRITEPROTECT is not set
# CONFIG_MMCSD_MMCSUPPORT is not set
# CONFIG_NSH_ARGCAT is not set
# CONFIG_NSH_CMDOPT_HEXDUMP is not set
# CONFIG_NSH_CMDPARMS is not set
# CONFIG_SPI_EXCHANGE is not set
CONFIG_ARCH="xtensa"
CONFIG_ARCH_BOARD="esp32-wrover-kit"
CONFIG_ARCH_BOARD_COMMON=y
CONFIG_ARCH_BOARD_ESP32_WROVERKIT=y
CONFIG_ARCH_BUTTONS=y
CONFIG_ARCH_CHIP="esp32"
CONFIG_ARCH_CHIP_ESP32=y
CONFIG_ARCH_CHIP_ESP32WROVER=y
CONFIG_ARCH_IRQBUTTONS=y
CONFIG_ARCH_STACKDUMP=y
CONFIG_ARCH_XTENSA=y
CONFIG_BOARD_LOOPSPERMSEC=16717
CONFIG_BUILTIN=y
CONFIG_DEBUG_ERROR=y
CONFIG_DEBUG_FEATURES=y
CONFIG_DEBUG_INFO=y
CONFIG_DEBUG_MEMCARD=y
CONFIG_DEBUG_MEMCARD_ERROR=y
CONFIG_DEBUG_MEMCARD_INFO=y
CONFIG_DEBUG_MEMCARD_WARN=y
CONFIG_DEBUG_WARN=y
CONFIG_ESP32_SPI2=y
CONFIG_ESP32_SPI2_CSPIN=13
CONFIG_ESP32_SPI2_MISOPIN=2
CONFIG_ESP32_SPI2_MOSIPIN=15
CONFIG_ESP32_UART0=y
CONFIG_EXAMPLES_BUTTONS=y
CONFIG_EXAMPLES_LEDS=y
CONFIG_EXAMPLES_LUA_MODULE=y
CONFIG_FAT_LCNAMES=y
CONFIG_FAT_LFN=y
CONFIG_FS_FAT=y
CONFIG_FS_PROCFS=y
CONFIG_HAVE_CXX=y
CONFIG_HAVE_CXXINITIALIZE=y
CONFIG_IDLETHREAD_STACKSIZE=3072
CONFIG_INIT_ENTRYPOINT="nsh_main"
CONFIG_INPUT=y
CONFIG_INPUT_BUTTONS=y
CONFIG_INPUT_BUTTONS_LOWER=y
CONFIG_INTELHEX_BINARY=y
CONFIG_INTERPRETERS_LUA=y
CONFIG_MMCSD=y
CONFIG_MMCSD_SPICLOCK=4000000
CONFIG_MM_REGIONS=3
CONFIG_NSH_ARCHINIT=y
CONFIG_NSH_BUILTIN_APPS=y
CONFIG_NSH_FILEIOSIZE=512
CONFIG_NSH_LINELEN=64
CONFIG_NSH_MMCSDSPIPORTNO=2
CONFIG_NSH_READLINE=y
CONFIG_PREALLOC_TIMERS=4
CONFIG_RAM_SIZE=114688
CONFIG_RAM_START=0x20000000
CONFIG_RR_INTERVAL=200
CONFIG_START_DAY=6
CONFIG_START_MONTH=12
CONFIG_START_YEAR=2011
CONFIG_SYSTEM_NSH=y
CONFIG_UART0_SERIAL_CONSOLE=y
CONFIG_USERLED=y
CONFIG_USERLED_LOWER=y
```

Reply:
The first config worked here (has run fine).
I have executed `./tools/refresh.sh esp32-wrover-kit:lua` and commited the changes to the PR.
The one that is now committed works fine with LUA / WroverKit.

Reply:
Nice! If the CI build passes without any errors, squash the commits and you are good to go 👍🏼

Reply:
Also, when squashing, I suggest making two different commits:

- boards/esp32-wrover-kit: Fix GPIO conflict
- boards/esp32-wrover-kit: Add Lua Interpreter defconfig

Reply:
The very first commit already includes all files... 
I can squash them all, but I think that I can't break a commit into 2 different commits.

Reply:
I'll rebase it and commit it again in 2 pieces.